### PR TITLE
Added : Pop My Baloon #3718

### LINKED ISSUE #3178: [New game]:  POP the baloon with your mouse cursor
### 🎮 Game Request

Game logic and basic description : It's a baloon pop  Game, made using HTML,CSS,JS . The Code is Responsive and here Player has given some rando baloons in their screen  and he/she has to pop the baloons
@kunjgit Sir Kindly Assign me this Project !!!

### Point down the features

**Tech Stack: HTML,CSS,JS**

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by codewithganeshhh**
## PR Description 📜

I added pop my baloon game with readme file with full introduction of game . It was a single player game with mouse cursor contol. 

Fixes  #3178 

<hr>
 
## Mark the task you have completed ✅

- [✅ ] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [✅ ] I have performed a self-review of my own code or work.
- [✅] I have commented my code, particularly in hard-to-understand areas.
- [✅ ] My changes generates no new warnings.
- [✅ ] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [✅ ] I have added screenshot for website preview in assets/images 
- [ ✅] I have added entries for my game in main README.md
- [✅ ] I have added README.md in my folder 
- [✅ ] I have added working video of the game in README.md (optional)
- [✅ ] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

Pop The Baloons

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @codewithganeshhh ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: assets/index_old.html [hunk 1]
```diff
@@ -2618,6 +2618,20 @@ <h3 class="project-title"><a href="https://github.com/kunjgit/GameZone/tree/main
...
+               
+              <img src="\GameZone\Pop The Baloons.png" alt="game thumbnail" loading="lazy">
+            </figure>
+            <h3 class="project-title"><a href="https://github.com/codewithganeshhh/GameZone/blob/Work/Games/Pop%20The%20Baloon/index.html"
+                target="_blank">137.Pop The Baloon 🔗</a></h3>
+            <p class="project-category">Play and have fun!</p>
+          </a>
+        </li>
+
+
```
Comment (kunjgit) [inline assets/index_old.html:2635]: @codewithganeshhh no need to change it over here !
this is older html file just for reference !
make sure you are adding your game in main README file !

**Review by kunjgit — CHANGES_REQUESTED** [review]

File: assets/index_old.html (hunk 1 above)
Comment (codewithganeshhh) [reply assets/index_old.html:2635]: yes i added

**Review by kunjgit — CHANGES_REQUESTED** [review]
@codewithganeshhh you have removed index_old which is for future reference make sure you are not having any changes in that but keep that file there !

**Comment by kunjgit** [comment]
Thank you  @codewithganeshhh , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**